tapir.json

```diff
@@ -0,0 +1,42 @@
+{
+    "11155111": [
+        "https://rpc.sepolia.org",
+        "https://rpc2.sepolia.org",
+        "https://rpc-sepolia.rockx.com",
+        "https://rpc.sepolia.ethpandaops.io",
+        "https://sepolia.gateway.tenderly.co",
+        "https://ethereum-sepolia-rpc.publicnode.com",
+        "https://sepolia.drpc.org"
+    ],
+    "80002": [
+        "https://rpc-amoy.polygon.technology",
+        "https://polygon-amoy-bor-rpc.publicnode.com"
+    ],
+    "84532": [
```

Comment:
For `tapir` let's only include sepolia and polygon amoy for now i.e. let's remove the other chains. We'll only focus on `lynx` for now, for "any chain" functionality.

Reply:
done :)